homeassistant/components/motioneye/__init__.py

```diff
@@ -97,6 +175,12 @@ def listen_for_new_cameras(
     )
 
 
+async def async_setup(hass: HomeAssistant, config: dict[str, Any]) -> bool:
```

Comment:
This is the place where `ConfigType` can be used.
```suggestion
async def async_setup(hass: HomeAssistant, config: ConfigType) -> bool:
```